### DISCUSSION TITLE
[Feature] Manually tag faces and objects, search in description

### DISCUSSION THREAD

**Discussion by sandroshu** [immich-app/immich#5245] [Feature Request]
### The feature

It would be nice if we could manually tag faces and other objects on the uploaded pictures.
In some cases the face/object decetion does not detect everything. In general it is fine, but there are some cases when it would be very useful.

Currently if I add a description to a photo and I query it in search it does not show up as a result. It would be engough to fulfill this request to also include photos with description matching the search term, however the best possible solution would be to include the manually tagged photo when searching for a specific person/object and also considering the description.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of https://github.com/immich-app/immich/discussions/2472#discussioncomment-6850486
